data.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES
+(1,' Agumon', '2020-02-03', 0, TRUE, 10.23),
+(2,'Gabumon', '2018-11-15', 2, TRUE, 8),
+(3,'Pikachu', '2021-01-07', 1, FALSE, 15.04),
+(4,'Devimon', '2017-05-12', 5, TRUE, 11);
```

Comment:
- [ ] Kindly remove the id field as it should be passed as an autoincrement. Setting it as autoincrement on the schema.sql does not require you to pass an ID for any animal being created. Manually setting the id is not idealistic and hence, we set the ID on the schema as an autoincrement field. Kindly work on that👍🏽

Reply:
Thank you so much for you review @AmaduKamara , I have made the suggested changes.